nomic/project.py

```diff
@@ -1370,7 +1006,9 @@ def create_index(
                         reuse_embedding_from_index_id = index.id
                         break
                 if reuse_embedding_from_index_id is None:
-                    raise Exception(f"Could not find the index '{reuse_embeddings_from_index}' to re-use from. Possible options are {[index.name for index in indices]}")
+                    raise Exception(
+                        f"Could not find the index '{reuse_embeddings_from_index}' to re-use from. Possible options are {[index.name for index in indices]}"
+                    )
```

Comment:
Seeing how aggressively we use the word 'Index' here, I'm wondering if we should make atlas.maps return all the indices, rather than the projections.... I think it will be confusing for people to encounter the idea of an index if we're calling them 'maps'

Reply:
It is certainly confusing to have many abstraction, we have talked in person about this in-depth before.

Currently AtlasIndex is a proxy container from the ages where we were not certain what Atlas would be so built in room for flexibility.

I would propose we just *ignore* it and only mention AtlasProject and AtlasProjection ("map") across all documentation. We could essentially replace the word index to map in every location and no-one would be the wiser (and the code may be the clearer) as AtlasIndex is currently a passthrough object.

Reply:
That sounds good to me--so this method will be renamed `reuse_embeddings_from_map`?